index.html

```diff
@@ -12,6 +12,46 @@
     <link href="https://fonts.googleapis.com/css2?family=Nunito:wght@400;900&display=swap" rel="stylesheet">
 </head>
 <body>
-    <div class="container"></div>
+    <div class="container">
+        <h3>This is a card structure, inspired by one of the codepen samples but I have designed the layout by myself</h3>
+        <div class="card">
+            <div class="top">
+                <img src="images/nike-shoe.png" class="product-image">
+            </div>
+            <div class="bottom">
+                <div class="product-info">
+                    <div class="product-name">
+                        <h3 class="product-type">Hiking Shoes</h3>
+                    </div>
+                    <div class="product-details">
+                        <h3 class="product-heading">Mountain Collection</h3>
+                        <h2 class="product-description">Men High Altitude</h2>
```

Comment:
Please refrain from using heading tags to make text look big. Heading tags should only be used for heading purpose. You can style this in a span tag.
Is there a specific reason to use h2 here? Let me know your thoughts about this.